shopfloor_mobile_base/static/wms/src/services/config_registry.js

```diff
@@ -20,6 +20,7 @@ class StoredConfig {
         this.key = key;
         this.default = meta.default;
         this.reset_on_clear = meta.reset_on_clear;
+        this.storage_driver = meta.storage_driver;
```

Comment:
what if here we look for a dict?
We could have different TTL  per key.
```suggestion
        this.storage = meta.storage || {};
        [...]
        this.storage.driver | ttl | whatever
```

Reply:
and in the app init we could consider getting these defaults from `shopfloor_app_info`